www/src/data/sidebars/tutorial-links.yaml

```diff
@@ -10,8 +10,8 @@
       items:
         - title: Familiarize with the command line
           link: /tutorial/part-zero/#familiarize-yourself-with-the-command-line
-        - title: Install Node.js and npm
-          link: /tutorial/part-zero/#-install-nodejs-and-npm
+        - title: Install Node.js
```

Comment:
I'm not sure if we need to change the title. `npm` is distributed with `Node.js`, which means that it automatically gets installed on your machine while installing `Node.js` and we might want to be clear about that and we might need to update the docs as well.

Reply:
The title of the section was changed from "Install Node.js and npm" to "Install Node.js for your appropriate operating system". If anything, it should reflect the actual title of the section.